packages/app-desktop/gui/NoteEditor/NoteTitle/NoteTitleBar.tsx

```diff
@@ -91,7 +91,7 @@ export default function NoteTitleBar(props: Props) {
 	}, []);
 
 	function renderTitleBarDate() {
-		return <span style={styles.titleDate}>{time.formatMsToLocal(props.noteUserUpdatedTime)}</span>;
+		return <span className="user-updated-time" style={styles.titleDate}>{time.formatMsToLocal(props.noteUserUpdatedTime)}</span>;
```

Comment:
To name the classes, could you follow the pattern "something-type", for example as you've named "tag-bar", or "title-input". So here it could be `updated-time-label` for example.